src/services/Twitter.js

```diff
@@ -17,6 +17,52 @@ const clientTwitter = new Twit({
   access_token_secret: TWITTER_ACCESS_TOKEN_SECRET,
 });
 
+/**
+* Post thread, each tweet can have text and photos or only text
+*
+* @async
+* @param {Array<Object>} tweetSeq
+*/
+const uploadThreadTwitter = (tweetSeq, tweetId = '') => {
+  if (tweetSeq.length === 0) {
+    return;
+  }
+
+  const tweetToSend = tweetSeq.shift();
+
+  const uploadedMedia = tweetToSend.media !== undefined
+    ? tweetToSend.media.map((filedata) => {
+      const fileContent = isBase64(filedata)
+        ? filedata
+        : fs.readFileSync(`${path.resolve('./src/images')}${path.sep}${filedata}`, { encoding: 'base64' });
+
+      return clientTwitter.post('media/upload', { media_data: fileContent });
+    })
+    : [];
+
+  Promise.all(uploadedMedia).then((results) => {
+    const mediaIds = results.map(({ data }) => {
+      const { media_id_string: mediaId } = data;
+
+      clientTwitter.post('media/metadata/create', { media_id: mediaId });
+
+      return mediaId;
+    });
+
+    const params = {
+      status: tweetToSend.status,
+      media_ids: mediaIds,
+      in_reply_to_status_id: tweetId,
```

Comment:
What'll happen if `tweetId`is `''` (as if we don't pass that argument)?

Reply:
Twitter API will receive an empty string in `in_reply_to_status_id` tweet parameter, and therefore will ignore it